chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md

```diff
@@ -80,12 +80,12 @@ and then follow the directions given by the other, as is seen in :numref:`fig_ad
 Vector subtraction has a similar interpretation.
 By considering the identity that $\mathbf{u} = \mathbf{v} + (\mathbf{u}-\mathbf{v})$,
 we see that the vector $\mathbf{u}-\mathbf{v}$ is the direction 
-that takes us from the point $\mathbf{u}$ to the point $\mathbf{v}$.
+that takes us from the point $\mathbf{v}$ to the point $\mathbf{u}$.
 
 
 ## Dot Products and Angles
 As we saw in :numref:`sec_linear-algebra`, 
-if we take two column vectors say $\mathbf{u}$ and $\mathbf{v}$,
+if we take two column vectors saying $\mathbf{u}$ and $\mathbf{v}$,
```

Comment:
This change is also not grammatical. Delete the word "say" as it was superfluous.  So it should read "if we take two column vectors $\mathbf{u}$ and $\mathbf{v}$,"

Reply:
Ok!